test/functional/content_view/version/incremental_update_test.rb

```diff
@@ -88,4 +88,21 @@
     result = run_cmd(@cmd + params)
     assert_equal(result.exit_code, 0)
   end
+
+  it "performs incremental update with packages" do
+    params = ['--content-view-version-id=5', '--packages=bla', '--lifecycle-environment-ids=1']
+
+    expect_generic_search(
+      :packages, params: {search: "name = \"bla\""}, returns: {'id' => 5}
```

Comment:
This test would be more clear if the package ID didn't match the content_view_version_id. Just pick a different number.